libpod/runtime.go

```diff
@@ -157,6 +163,15 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 		}
 	}
 
+	// Set up the CNI net plugin
```

Comment:
Should we not do this if netmode.IsHost()

Reply:
I think it might be worth adding a WithDisableNetworking() to turn off networking in the runtime for unit tests and kpod run --net=host, but I think we can get this in without it